learn/advanced/asynchronous_operations.mdx

```diff
@@ -126,18 +126,24 @@ When you make a [request for an asynchronous operation](#which-operations-are-as
 2. When your task reaches the front of the queue, Meilisearch begins working on it. Task `status` set to `processing`
 3. Meilisearch finishes the task. Status set to `succeeded` if task was successfully processed, or `failed` if there was an error
 
-### Canceling and deleting tasks
+<Capsule intent="note">
+**Terminating a Meilisearch instance in the middle of an asynchronous operation is completely safe** and will never adversely affect the database.
+</Capsule>
 
-You can cancel a task while it is `enqueued` or `processing` by using [the cancel tasks endpoint](/reference/api/tasks#cancel-tasks). Doing so changes a task's `status` to `canceled`.
+### Canceling tasks
 
-Meilisearch does not automatically delete tasks once their status is `succeeded`, `failed`, or `canceled`. These tasks remain visible in [the task list](/reference/api/tasks#get-tasks). To delete them, use the [delete tasks route](/reference/api/tasks#delete-tasks).
+You can cancel a task while it is `enqueued` or `processing` by using [the cancel tasks endpoint](/reference/api/tasks#cancel-tasks). Doing so changes a task's `status` to `canceled`.
 
 <Capsule intent="note">
-**Terminating a Meilisearch instance in the middle of an asynchronous operation is completely safe** and will never adversely affect the database.
-
 Tasks are not canceled when you terminate a Meilisearch instance. Meilisearch discards all progress made on `processing` tasks and resets them to `enqueued`. Task handling proceeds as normal once the instance is relaunched.
 </Capsule>
 
+### Deleting tasks
+
+Completed tasks remain visible in [the task list](/reference/api/tasks#get-tasks). To delete them manually, use the [delete tasks route](/reference/api/tasks#delete-tasks).
```

Comment:
Since we never really define what « completed » or « finished » means, I think it could be nice to list it what do you think?

```suggestion
Finished tasks (succeeded, failed, or canceled) remain visible in [the task list](/reference/api/tasks#get-tasks). To delete them manually, use the [delete tasks route](/reference/api/tasks#delete-tasks).
```